lambda/MAS.Tests/Infrastructure/Extensions.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Shouldly;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+
+namespace MAS.Tests.Infrastructure
+{
+    public static class Extensions
+    {
+        public static void ShouldMatchApproved(this string content,
+                [System.Runtime.CompilerServices.CallerMemberName] string memberName = null,
+                [System.Runtime.CompilerServices.CallerFilePath] string sourceFilePath = null)
+        {
+         
+            var expectedFileDir = Path.GetDirectoryName(sourceFilePath);
+            var callingFileNameWithoutExtension = Path.GetFileNameWithoutExtension(sourceFilePath);
+            var expectedFileName = $"{callingFileNameWithoutExtension}.{memberName}.approved.txt";
+
+            var expectedFilePath = Path.Combine(expectedFileDir, expectedFileName);
+
+            if (File.Exists(expectedFilePath))
+            {
+                var expectedText = File.ReadAllText(expectedFilePath, Encoding.UTF8);
+
+                var normalisedExpectedText = expectedText.Replace("\r", "");//normalising line endings.
+                var normalisedContent = content.Replace("\r", "");
+
+                if (normalisedExpectedText.Equals(normalisedContent))
```

Comment:
You can make it short hard:
if (normalisedExpectedText.Equals(normalisedContent))
    return;